### PR TITLE
Add urdf parser

### PR DESCRIPTION
**Pull request by tkoolen**
Currently able to handle spheres, boxes, and cylinders (requires https://github.com/rdeits/DrakeVisualizer.jl/pull/3).

What are your thoughts on testing this? Do you want to include a couple of example URDFs?

### CODE REVIEW

File: src/parse_urdf.jl [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    xml_materials = get_elements_by_tagname(xroot, "material")
+    named_colors = [attribute(m, "name")::ASCIIString => parse_material(Float64, m)::RGBA{Float64} for m in xml_materials]
+    bods = bodies(mechanism)
+    geometry_data = GeometryData[]
+    vis_data = Link[]
+    for xml_link in xml_links
+        xml_visuals = get_elements_by_tagname(xml_link, "visual")
+        linkname = attribute(xml_link, "name")
+        body = bods[findfirst(b -> RigidBodyDynamics.name(b) == linkname, bods)]
+        if !isroot(body) # geometry attached to world is currently not supported. Also, this is the only reason to have the Mechanism argument for this function
```
Comment (tkoolen) [inline src/parse_urdf.jl]: note this comment.

**Comment by tkoolen** [comment]
Just realized this requires me to merge some changes in RigidBodyDynamics.jl too; I'll let you know when that's done.

**Comment by tkoolen** [comment]
Done.

**Comment by rdeits** [comment]
Yeah, I think having sample URDFs is great. If you end up wanting to include samples with RigidBodyDynamics, then we might someday want to create a separate package for them, but I don't mind having a few in this repo anyway. Got any favorites?

**Comment by rdeits** [comment]
Also, I think I can add mesh parsing support pretty easily. I was reluctant to do it just to draw inertial ellipsoids, but being able to load the full Atlas or Val mesh would be pretty cool. I think MeshIO can pretty much just do it for us.

**Pull request merged**